From d00f9f9af1585e9b4de1c25b94bd9dda803dac0a Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 26 Jun 2014 14:25:20 -0400
Subject: [Documentation] MAINTAINERS: Add DesignWare, i.MX6, Armada, R-Car PCI host maintainers

Message-id: <20140626142520.18681.22261.stgit@gir.stowe>
Patchwork-id: 84970
O-Subject: [RHEL7.1 PATCH 7/7] MAINTAINERS: Add DesignWare, i.MX6, Armada, R-Car PCI host maintainers
Bugzilla: 1110890
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110890
Upstream Status: f0b75693cbb26439ba959ba7d3b4f43e2fcf3da6

commit f0b75693cbb26439ba959ba7d3b4f43e2fcf3da6
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Wed Dec 11 10:40:59 2013 -0700

    MAINTAINERS: Add DesignWare, i.MX6, Armada, R-Car PCI host maintainers

    Add entries for PCI host controller drivers in drivers/pci/host/.

    Signed-off-by: Mohit Kumar <mohit.kumar@st.com>             # DESIGNWARE
    Signed-off-by: Pratyush Anand <pratyush.anand@st.com>       # DESIGNWARE
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Acked-by: Simon Horman <horms+renesas@verge.net.au>         # R-CAR
    Double-Plus-Acked-by: Jason Cooper <jason@lakedaemon.net>   # MVEBU
    Acked-by: Jingoo Han <jg1.han@samsung.com>
    Acked-by: Thierry Reding <treding@nvidia.com>
    Acked-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
    Acked-by: Richard Zhu <r65037@freescale.com>                # IMX6

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/MAINTAINERS b/MAINTAINERS
index c8ad49f..59c6a2e 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -6200,19 +6200,52 @@ F: drivers/pci/
 F: include/linux/pci*
 F: arch/x86/pci/
 
+PCI DRIVER FOR IMX6
+M: Richard Zhu <r65037@freescale.com>
+M: Shawn Guo <shawn.guo@linaro.org>
+L: linux-pci@vger.kernel.org
+L: linux-arm-kernel@lists.infradead.org (moderated for non-subscribers)
+S: Maintained
+F: drivers/pci/host/*imx6*
+
+PCI DRIVER FOR MVEBU (Marvell Armada 370 and Armada XP SOC support)
+M: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
+M: Jason Cooper <jason@lakedaemon.net>
+L: linux-pci@vger.kernel.org
+L: linux-arm-kernel@lists.infradead.org (moderated for non-subscribers)
+S: Maintained
+F: drivers/pci/host/*mvebu*
+
 PCI DRIVER FOR NVIDIA TEGRA
 M: Thierry Reding <thierry.reding@gmail.com>
 L: linux-tegra@vger.kernel.org
+L: linux-pci@vger.kernel.org
 S: Supported
 F: Documentation/devicetree/bindings/pci/nvidia,tegra20-pcie.txt
 F: drivers/pci/host/pci-tegra.c
 
+PCI DRIVER FOR RENESAS R-CAR
+M: Simon Horman <horms@verge.net.au>
+L: linux-pci@vger.kernel.org
+L: linux-sh@vger.kernel.org
+S: Maintained
+F: drivers/pci/host/*rcar*
+
 PCI DRIVER FOR SAMSUNG EXYNOS
 M: Jingoo Han <jg1.han@samsung.com>
 L: linux-pci@vger.kernel.org
+L: linux-arm-kernel@lists.infradead.org (moderated for non-subscribers)
+L: linux-samsung-soc@vger.kernel.org (moderated for non-subscribers)
 S: Maintained
 F: drivers/pci/host/pci-exynos.c
 
+PCI DRIVER FOR SYNOPSIS DESIGNWARE
+M: Mohit Kumar <mohit.kumar@st.com>
+M: Jingoo Han <jg1.han@samsung.com>
+L: linux-pci@vger.kernel.org
+S: Maintained
+F: drivers/pci/host/*designware*
+
 PCMCIA SUBSYSTEM
 P: Linux PCMCIA Team
 L: linux-pcmcia@lists.infradead.org
-- 
1.7.1